applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -148,30 +155,32 @@ static void cap_start_worker(struct k_work *work)
 	param.count = 0;
 	param.type = BT_CAP_SET_TYPE_AD_HOC;
 
```

Comment:
```suggestion
struct le_audio_unicast_server *unicast_server = &unicast_servers[lvl1_idx][lvl2_idx][lvl3_idx];
```
As before I think this abstraction makes the code less intimidating to read as it would replace `unicast_servers[lvl1_idx][lvl2_idx][lvl3_idx]` multiple places in this function

Reply:
yeah, it makes sense some places where there is no iteration over different unicast_servers. Can swap those I find out with this